Source/Core/Core/PowerPC/JitArm64/JitArm64_SystemRegisters.cpp

```diff
@@ -231,8 +234,8 @@ void JitArm64::twx(UGeckoInstruction inst)
     CMP(gpr.R(a), gpr.R(inst.RB));
   }
 
-  std::vector<FixupBranch> fixups;
-  CCFlags conditions[] = {CC_LT, CC_GT, CC_EQ, CC_VC, CC_VS};
+  constexpr std::array<CCFlags, 5> conditions{{CC_LT, CC_GT, CC_EQ, CC_VC, CC_VS}};
+  Common::SmallVector<FixupBranch, 5> fixups;
 
   for (int i = 0; i < 5; i++)
```

Comment:
[Opinion] I would replace the 5 in the loop index with conditions.size(), and probably in the declaration of fixups too.